services/compileGRPCStubs/index.ts

```diff
@@ -118,6 +118,36 @@ const zippedServiceFilePath = (orgId: string, fileName: string): string => {
   return `${directory}/${fileName}`;
 };
 
+const findFile = (files: string[], pattern: string): string => {
+  for (const file of files) {
+    if (file.includes(pattern)) {
+      return file;
+    }
+  }
+  throw "File not found";
+};
+
+const findGRPCFile = (files: string[]): string => {
+  return findFile(files, "grpc_pb.js");
+};
+
+const findServiceFile = (files: string[]): string => {
+  return findFile(files, "service_pb.js");
```

Comment:
As per my knowledge, the name `service` might not be there in all the files. 
Are you sure about this. Can you validate this once again.

Reply:
Fix in the latest commit